linkerd/app/core/src/telemetry/build_info.rs

```diff
@@ -1,14 +1,11 @@
 use linkerd_metrics::{metrics, FmtLabels, FmtMetric, FmtMetrics, Gauge};
-use std::env;
-use std::fmt;
-use std::string::String;
-use std::sync::Arc;
+use std::{env, fmt};
 
-const GIT_BRANCH: &str = env!("GIT_BRANCH");
-const GIT_SHA: &str = env!("GIT_SHA");
-const GIT_VERSION: &str = env!("GIT_VERSION");
-const PROFILE: &str = env!("PROFILE");
-const RUST_VERSION: &str = env!("RUST_VERSION");
```

Comment:
any motivation for removing the rust compiler information from the metric? i know we don't really use it very often (especially because a proxy tag will include a `.rust-version.toml`), but it seems more or less free to include regardless?

Reply:
I don't think it's useful information to Linkerd operators.